ruby/ql/lib/codeql/ruby/Regexp.qll

```diff
@@ -5,8 +5,8 @@
  * terms.
  */
 
-import regexp.RegExpTreeView // re-export
-private import regexp.ParseRegExp
+import regexp.internal.RegExpTreeView // re-export
```

Comment:
This appears dangerous: I would expect I could make all sorts of breaking changes to a file in an `internal` folder (except perhaps if it contains a `Public` module that documents it explicitly).